subworkflows/local/bnd_benchmark/main.nf

```diff
@@ -0,0 +1,99 @@
+include { BCFTOOLS_REHEADER as BCFTOOLS_REHEADER_3  } from '../../../modules/local/bcftools/reheader'
+include { BCFTOOLS_REHEADER as BCFTOOLS_REHEADER_4  } from '../../../modules/local/bcftools/reheader'
+
+workflow BND_BENCHMARK {
+    take:
+    input_ch  // channel: [val(meta), test_vcf, test_index, truth_vcf, truth_index, regions_bed, targets_bed]
+    fai       // reference channel [val(meta), ref.fa.fai]
+
+    main:
+
+    versions        = Channel.empty()
+    tagged_variants = Channel.empty()
+
+    RTGTOOLS_BNDEVAL(
+        input_ch.map{ meta, vcf, _tbi, _truth_vcf, _truth_tbi, _regionsbed, _targets_bed  ->
+                    [ meta, vcf, _tbi, _truth_vcf, _truth_tbi, _regionsbed ]
+                }
+    )
+    versions = versions.mix(RTGTOOLS_BNDEVAL.out.versions)
+
+    RTGTOOLS_BNDEVAL.out.summary
+            .map { _meta, file -> tuple([vartype: params.variant_type] + [benchmark_tool: "rtgtools"], file) }
+            .groupTuple()
+            .set { summary_reports }
+
+
+    // reheader benchmarking results properly and tag meta
+    BCFTOOLS_REHEADER_1(
+        RTGTOOLS_BNDEVAL.out.fn_vcf.map{ meta, vcf ->
+        [ meta, vcf, [], [] ]
+        },
+        fai
+    )
+    versions = versions.mix(BCFTOOLS_REHEADER_1.out.versions.first())
+
+    BCFTOOLS_REHEADER_1.out.vcf
+        .join(BCFTOOLS_REHEADER_1.out.index)
+        .map { _meta, file, index -> tuple([vartype: params.variant_type] + [tag: "FN"] + [id: "rtgtools"], file, index) }
+        .set { vcf_fn }
+
+    BCFTOOLS_REHEADER_2(
+        RTGTOOLS_BNDEVAL.out.fp_vcf.map{ meta, vcf ->
+        [ meta, vcf, [], [] ]
+        },
+        fai
+    )
+    versions = versions.mix(BCFTOOLS_REHEADER_2.out.versions)
+
+    BCFTOOLS_REHEADER_2.out.vcf
+        .join(BCFTOOLS_REHEADER_2.out.index)
+        .map { _meta, file, index -> tuple([vartype: params.variant_type] + [tag: "FP"] + [id: "rtgtools"], file, index) }
+        .set { vcf_fp }
+
+    BCFTOOLS_REHEADER_3(
+        RTGTOOLS_BNDEVAL.out.baseline_vcf.map{ meta, vcf ->
+        [ meta, vcf, [], [] ]
+        },
+        fai
+    )
+    versions = versions.mix(BCFTOOLS_REHEADER_3.out.versions)
+
+    BCFTOOLS_REHEADER_3.out.vcf
+        .join(BCFTOOLS_REHEADER_3.out.index)
+        .map { _meta, file, index -> tuple([vartype: params.variant_type] + [tag: "TP_base"] + [id: "rtgtools"], file, index) }
+        .set { vcf_tp_base }
+
+    BCFTOOLS_REHEADER_4(
+        RTGTOOLS_BNDEVAL.out.tp_vcf.map{ meta, vcf ->
+        [ meta, vcf, [], [] ]
+        },
+        fai
+    )
+    versions = versions.mix(BCFTOOLS_REHEADER_4.out.versions)
+
+    BCFTOOLS_REHEADER_4.out.vcf
+        .join(BCFTOOLS_REHEADER_4.out.index)
+        .map { _meta, file, index -> tuple([vartype: params.variant_type] + [tag: "TP_comp"] + [id: "rtgtools"], file, index) }
+        .set { vcf_tp_comp }
+
+    tagged_variants = tagged_variants.mix(
```

Comment:
just wanna make sure if thats an issue. Better to use .concat here? So Does order matter then use .concat.

Reply:
:) order is not important for this channel